serenata_toolbox/speeches.py

```diff
@@ -0,0 +1,88 @@
+import os
+import pandas as pd
+import numpy as np
+import xml.etree.ElementTree as ET
+import urllib
+
+from datetime import datetime
+
+class Speeches:
+
+    URL = 'http://www.camara.leg.br/SitCamaraWS/SessoesReunioes.asmx/ListarDiscursosPlenario?dataIni={dataIni}&dataFim={dataFim}&codigoSessao=&parteNomeParlamentar=&siglaPartido=&siglaUF='
+    FILE_BASE_NAME = 'speeches.xz'
+
+    CSV_PARAMS = {
+        'compression': 'xz',
+        'encoding': 'utf-8',
+        'index': False
+    }
+
+    def fetch(self, range_start, range_end):
+        range = {'dataIni': range_start, 'dataFim': range_end}
+        url = self.URL.format(**range)
+        file=urllib.request.urlopen(url)
+
+        t = ET.ElementTree(file=file)
+        records = []
+        root = t.getroot()
+        for i, session in enumerate(root):
```

Comment:
I might have missed it, but it looks like that you're not using the `i` (the index returned by the `enumerate`). Can we use just `for session in root:`?